app/helpers/protocols_io_helper.rb

```diff
@@ -145,4 +145,283 @@ def not_null(attribute)
   def close_open_html_tags(text)
     Nokogiri::HTML::DocumentFragment.parse(text).to_html
   end
+
+  def prepare_for_view(attribute_text, size)
```

Comment:
You could write this method in more compact way:
```
def prepare_for_view(attribute_text, size, table='no_table')
  attribute_text_not_null = not_null(attribute_text)
  attribute_text_not_null = string_html_table_remove(attribute_text_not_null) if table=='table_present'
  pio_eval_len(sanitize_input(attribute_text_not_null), size)
end
```
Then you would call the method like this:
`prepare_for_view(attribute_text, ProtocolsIoHelper::PIO_ELEMENT_RESERVED_LENGTH_SMALL, 'table_present')`